src/thunderbots/software/ai/hl/stp/tactic/block_shot_path_tactic.h

```diff
@@ -17,7 +17,8 @@ class BlockShotPathTactic : public Tactic
      * @param loop_forever Whether or not this Tactic should never complete. If true, the
      * tactic will be restarted every time it completes
      */
-    explicit BlockShotPathTactic(const Field& field, bool loop_forever = false);
+    // TODO: Should this loop forever???
```

Comment:
I think this is fine to have loop_forever by default, or you can remove the option entirely if it should _always_ run forever. In this case, I think that's fine just make sure to mention it in the javadoc comment.

In the future I'm considering not giving the `loop_forever` a default value so it's very explicit which it's doing. That way we avoid ambiguous usage in the `Play`

Reply:
I'm all for removing the default value

Reply:
Removed the default value.

Reply:
Removed the default value in `cherry_pick_tactic.h`